README.md

```diff
@@ -19,24 +19,24 @@ Right-click any web project in Solution Explorer and hit **Manage Client-side Li
 
 ![Context menu](art/context-menu-project.png)
 
-This will create a `library.json` file in the root of the project.
+This will create a `libman.json` file in the root of the project.
 
-## library.json
+## libman.json
 
 ### Context menu
-Right-click `library.json` in Solution Explorer to access commands that help managing the libraries.
+Right-click `libman.json` in Solution Explorer to access commands that help managing the libraries.
 
-![context menu library.json](art/context-menu-config.png)
+![context menu libman.json](art/context-menu-config.png)
 
 ### Intellisense
-Edit the library.json file to install libraries. Every time the file is saved, Visual Studio will install/restore the packages.
+Edit the libman.json file to install libraries. Every time the file is saved, Visual Studio will install/restore the packages.
 
-![library.json](art/library.json%20typing.gif)
+![libman.json](art/library.json%20typing.gif)
 
-See [library.json reference](https://github.com/aspnet/LibraryManager/wiki/library.json-reference) for more information.
+See [libman.json reference](https://github.com/aspnet/LibraryManager/wiki/library.json-reference) for more information.
```

Comment:
We should update the wiki page name and update this link

Reply:
I will log a bug to update the Wiki once this change is in.